pkg/drivemgr/drivemgr-update-history.md

```diff
@@ -0,0 +1,7 @@
+### drivemgr update history
+
+drivemgr have several implementations, file here to track changeset to create new
+git commit for proprietary code.
+
+2023/04/07  proprietary code change, tracked by internal number 25693.
+2023/05/18  remove empty image csi-baremetal-scheduler.
```

Comment:
The line starting with "2023/04/07" is for rel/objectscale-1.2 branch commit. In our master branch commits, we don't have the corresponding commit.
For the next line starting with "2023/05/18", it would be better for us to make the date here corresponding to the commit in the private code?

Reply:
that sounds good, I will fix them.